src/index.html

```diff
@@ -1,13 +1,63 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=""
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <section class="card" data-qa="card">
+      <div class="card__item-img">
+        <img
+          src="./images/imac.jpeg"
+          alt="Image of Apple iMac 27 inch"
+          class="card__item-img-size"
+        >
+      </div>
+
+      <div class="card__item-description">
+        <div class="card__item-name">
+          <div>Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</div>
+        </div>
+
+        <div class="card__item-code">
+          <div>Код товара: 195434</div>
+        </div>
+
+        <div class="card__item-review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <span class="card__item-review-text">Отзывов: 5</span>
+        </div>
+
+        <div class="card__item-price">
+          <span class="card__item-price-word">Цена:</span>
+          <span class="card__item-price-value">69 999 грн</span>
+        </div>
+      </div>
+
+      <div class="card__buy-button">
+        <a
+          href="#item-page"
+          class="card__buy-button-link"
```

Comment:
Sample: imagine your a-tag has span, would you name class like "card__buy-button-link-span" ?
But you're cool with not-nesting-element-in-element 👍 
Here you may describe class like card__link
```suggestion
          class="card__link"
```